scripts/install.ps1

```diff
@@ -9,6 +9,7 @@ $zip = "tilt-" + $version + ".zip"
 $extractDir = "tilt-" + $version
 $binDir = "$HOME\bin"
 $dest = "$binDir\tilt.exe"
+$verifyInstall = "tilt verify-install"
```

Comment:
dead code

Reply:
Oops I put that in there thinking I'd be using it with `Invoke-Command` or something, will remove!